spaceship/lib/spaceship/test_flight/build.rb

```diff
@@ -217,8 +217,7 @@ def submit_for_testflight_review!
         return if ready_to_test?
         return if approved?
 
-        build = find_app_store_connect_build
-        Spaceship::ConnectAPI::Base.client.post_beta_app_review_submissions(build_id: build["id"])
+        Spaceship::ConnectAPI::Base.client.post_beta_app_review_submissions(build_id: id)
```

Comment:
This also looks unused now as of https://github.com/fastlane/fastlane/commit/7b2d9ab0847d02f60966f2084120efc447f41d04#diff-fb9d332b3e3b1f3dfb3c5968c4f0c218R300